test/distributed/org/apache/cassandra/distributed/impl/Instance.java

```diff
@@ -631,6 +633,7 @@ public void startup(ICluster cluster)
                 StorageService.instance.registerDaemon(CassandraDaemon.getInstanceForTesting());
                 if (config.has(GOSSIP))
                 {
+                    MigrationCoordinator.setUptimeFn(() -> TimeUnit.NANOSECONDS.toMillis(nanoTime() - startedAt));
```

Comment:
spoke in slack, can you also fix the startedAt bug?  This is object init, which != uptime.

When we run for 1m and restart, the restart should have uptime=0 but will be uptime=1m due to the fact we don't cleanup the object.